bin/worker.js

```diff
@@ -172,7 +172,6 @@ async function main () {
 
   config.scheduler =
     new taskcluster.Scheduler({ credentials: config.taskcluster });
-  config.schema = require('../lib/schema')();
```

Comment:
This should be something like:
```js
config.validator = await base.validator();
config.validator.register(require('../schemas/payload')); // or whereever its...
```
This way we only create the validator object once... And in a future where we have more than one schema we can load them globally...